portability-spi-transfer/src/main/java/org/datatransferproject/spi/transfer/idempotentexecutor/IdempotentImportExecutor.java

```diff
@@ -16,94 +16,46 @@
 
```

Comment:
(same) skipping review as I think this belongs to #1079 (feel free to merge in either - but I only reviewed the other copy)